helix-term/src/commands.rs

```diff
@@ -1019,6 +1019,34 @@ mod cmd {
         doc.format(view.id)
     }
 
+    fn earlier(editor: &mut Editor, args: &[&str], event: PromptEvent) {
+        let sotp = match args.join(" ").parse::<helix_core::StepsOrTimePeriod>() {
+            Ok(sotp) => sotp,
+            Err(msg) => {
+                editor.set_error(msg);
+                return;
+            }
+        };
+        let view_id = editor.view().id;
+        let doc_id = editor.view().doc;
+        let doc = &mut editor.documents[doc_id];
```

Comment:
```suggestion
        let (view, doc) = editor.current();
```
There is a helper function for this.

Reply:
I incorporated the change in my fork. I'm not sure if I want to commit from here. Will it change my repo?

Reply:
You don’t need to commit from here if you manually added the changes to your fork.

Reply:
Thanks 👍